### PR TITLE
Fix variable assignment

### PR DESCRIPTION
**Pull request by ricardoasmarques**
Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fix variable assignment (PR #9)
```
Comment (smithfarm) [inline CHANGELOG.md]: The changelog intro says: "All *notable* changes to this project will be documented in this file." (emphasis mine)

But "fix variable assignment" doesn't sound very "notable" - I mean from the perspective of someone reading the changelog. It sounds like something was fixed "under the hood", so not particularly notable for end users.

**Comment by smithfarm** [comment]
Oh, also, the changelog mod in this PR will conflict with the one in https://github.com/rjfd/sesdev/pull/8 (the one in #8 sounds more "notable" than this one).

File: CHANGELOG.md (hunk 1 above)
Comment (rjfd) [reply CHANGELOG.md]: Indeed. Thanks for pointing that out.

**Comment by rjfd** [comment]
PR #11 includes this fix, closing this PR

**Pull request closed without merging**